swri_roscpp/CMakeLists.txt

```diff
@@ -16,7 +16,28 @@ find_package(std_srvs REQUIRED)
 
 # TODO pjr What to do about swri_roscpp-extras.cmake?
 
-### Build Test Node ###
+add_library(${PROJECT_NAME} INTERFACE)
```

Comment:
This will cause a cmake error if tests are not disabled; a target named `${PROJECT_NAME}` is already defined below at line 105 with `rosidl_generate_interfaces(${PROJECT_NAME}`.  If this needs to be built and installed as a library, I would recommend renaming the test target.